{
  try {
    return Check.check(ctx.context,path);
  }
 catch (  final IOException ex) {
    Util.debug(ex);
    if (err)     (col ? NOCOLL : IOERR).thrw(ii,ex);
    return null;
  }
}
